src/custom/utils/xdai/hack.ts

```diff
@@ -4,6 +4,10 @@ import { ETHER, ChainId } from '@uniswap/sdk'
 
 let currentChainId: ChainId | undefined
 
+const XDAI_SYMBOL = 'xDai'
+const CURRENCY_SYMBOLS_XDAI = { ether: XDAI_SYMBOL, weth: 'WETH' }
```

Comment:
Is it intentional to have `weth` on xDai set to `WETH`?

Also, since we are here, another naming suggestion:

```
const CURRENCY_SYMBOLS_XDAI = { chainNative: XDAI_SYMBOL, chainWrapped: 'WETH' }
// OR
const CURRENCY_SYMBOLS_XDAI = { native: XDAI_SYMBOL, wrapped: 'WETH' }
```

Reply:
haha, not at all :) good catch

Reply:
Also, good suggestion in the name!